firmware/controllers/settings.cpp

```diff
@@ -1076,14 +1076,14 @@ static void setValue(const char *paramStr, const char *valueStr) {
 #if EFI_PROD_CODE
 	} else if (strEqualCaseInsensitive(paramStr, CMD_VSS_PIN)) {
 		setVssPin(valueStr);
-#endif // EFI_PROD_CODE
+#endif /* EFI_PROD_CODE */
```

Comment:
we have no consistency here :(

Reply:
IMO we should have no single line `/* */` comments anywhere